src/test/java/com/google/devtools/build/remote/client/DockerUtilTest.java

```diff
@@ -39,22 +38,28 @@ public void testGetDockerCommand() {
             .addArguments("escape<'>")
             .addEnvironmentVariables(
                 EnvironmentVariable.newBuilder().setName("PATH").setValue("/home/test"))
-            .build();
-    Action action =
-        Action.newBuilder()
-            .setCommandDigest(DIGEST_UTIL.compute(command.toByteArray()))
             .setPlatform(
                 Platform.newBuilder()
                     .addProperties(
                         Property.newBuilder()
                             .setName("container-image")
                             .setValue("docker://gcr.io/image")))
             .build();
-    String commandLine = DockerUtil.getDockerCommand(action, command, "/tmp/test");
-    assertThat(commandLine)
-        .isEqualTo(
-            "docker run -v /tmp/test:/tmp/test-docker -w /tmp/test-docker -e 'PATH=/home/test' "
-                + "gcr.io/image /bin/echo hello 'escape<'\\''>'");
+    String commandLine = DockerUtil.getDockerCommand(command, "/tmp/test");
+    long uid = DockerUtil.getUid();
```

Comment:
I'd rather you mocked it out instead of of running is -u from inside the test, but that would require a bit of restructuring. Up to you.